xml/System.CommandLine.Builder/CommandLineBuilderExtensions.xml

```diff
@@ -0,0 +1,700 @@
+             <code>
+                 &gt; myapp -a -b -c arg
+                 &gt; myapp -abc arg
+                 &gt; myapp -abcarg
+             </code></remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="ParseResponseFileAs">
+      <MemberSignature Language="C#" Value="public static System.CommandLine.Builder.CommandLineBuilder ParseResponseFileAs (this System.CommandLine.Builder.CommandLineBuilder builder, System.CommandLine.Parsing.ResponseFileHandling responseFileHandling);" />
+      <MemberSignature Language="ILAsm" Value=".method public static hidebysig class System.CommandLine.Builder.CommandLineBuilder ParseResponseFileAs(class System.CommandLine.Builder.CommandLineBuilder builder, valuetype System.CommandLine.Parsing.ResponseFileHandling responseFileHandling) cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.CommandLine.Builder.CommandLineBuilderExtensions.ParseResponseFileAs(System.CommandLine.Builder.CommandLineBuilder,System.CommandLine.Parsing.ResponseFileHandling)" />
+      <MemberSignature Language="VB.NET" Value="&lt;Extension()&gt;&#xA;Public Function ParseResponseFileAs (builder As CommandLineBuilder, responseFileHandling As ResponseFileHandling) As CommandLineBuilder" />
+      <MemberSignature Language="F#" Value="static member ParseResponseFileAs : System.CommandLine.Builder.CommandLineBuilder * System.CommandLine.Parsing.ResponseFileHandling -&gt; System.CommandLine.Builder.CommandLineBuilder" Usage="System.CommandLine.Builder.CommandLineBuilderExtensions.ParseResponseFileAs (builder, responseFileHandling)" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA;[System::Runtime::CompilerServices::Extension]&#xA; static System::CommandLine::Builder::CommandLineBuilder ^ ParseResponseFileAs(System::CommandLine::Builder::CommandLineBuilder ^ builder, System::CommandLine::Parsing::ResponseFileHandling responseFileHandling);" />
+      <MemberType>Method</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.CommandLine</AssemblyName>
+        <AssemblyVersion>2.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.CommandLine.Builder.CommandLineBuilder</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="builder" Type="System.CommandLine.Builder.CommandLineBuilder" RefType="this" />
+        <Parameter Name="responseFileHandling" Type="System.CommandLine.Parsing.ResponseFileHandling" />
+      </Parameters>
+      <Docs>
+        <param name="responseFileHandling">Specifies whether or how response files are parsed.</param>
+        <param name="builder">A command line builder.</param>
+        <summary>To be added.</summary>
+        <returns>The same instance of <see cref="T:System.CommandLine.Builder.CommandLineBuilder" />.</returns>
+        <remarks>To be added.</remarks>
+        <inheritdoc cref="P:System.CommandLine.Builder.CommandLineBuilder.ResponseFileHandling" />
```

Comment:
I don't think the `cref` attribute is currently supported in docs for `inheritdoc` - also not sure if you can inherit property doc comments for a method?